src/main/java/org/opentripplanner/api/model/ApiItineraryFares.java

```diff
@@ -0,0 +1,12 @@
+package org.opentripplanner.api.model;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+public record ApiItineraryFares(
```

Comment:
We need JavaDoc on the API types and on the properties(params).

Reply:
Docs added.